ntoskrnl/ob/oblife.c

```diff
@@ -1830,6 +1831,9 @@ NtSetInformationObject(IN HANDLE ObjectHandle,
 
         case ObjectSessionInformation:
 
+            /* Setup a lookup context */
+            ObpInitializeLookupContext(&LookupContext);
```

Comment:
Please move this to the code block where the locking is actually done (in the `NT_SUCCESS(Status)` one).
The variable declaration at the top can be moved there too.